src/gdb_main.c

```diff
@@ -668,7 +668,10 @@ static void handle_v_packet(char *packet, const size_t plen)
 			gdb_putpacketz("OK");
 
 	} else {
-		DEBUG_GDB("*** Unsupported packet: %s\n", packet);
+		/* The vMustReplyEmpty is used as a feature test to check how gdbserver handles unknown packets*/
+		/* print only if actually unknown */
+		if (strcmp(packet, "vMustReplyEmpty"))
```

Comment:
Would suggest adding the `!= 0` on the end to make it more explicit that we're testing for non-zero here.. but this works.

Reply:
Sure, that's a good thought